### ISSUE TITLE
Looses GI due to deprecated Scene loading

### ISSUE THREAD

**Issue by grayjamn** [BoltEngine/Bolt-Tracker#18]
###How to reproduce issue
1. Create two scene, add lights, and use Precomputed Realtime GI for one of the scenes (make sure objects are set to static)
2. In the first scene, use BoltNetwork.LoadScene() to load the second scene

Reasoning:
I believe I'm having an issue with how Bolt loads it scenes and my project's global illumination, although it's hard to be sure.

Basically, when switching between scenes, (in this case, my game's lobby to the actual level) I'm noticing I lose my scene's GI.

According to what I've read online, I think this issue is fixed in the latest version of Unity by the use of the SceneManager class. However, Bolt uses the old deprecated level loading system Application.LoadLevel within it's SceneLoader class.

Could I get this updated in Bolt to use the latest SceneManager class? You would probably want to have a check to see what version of Unity the user is using as SceneManager is a fairly recent addition to Unity.

###Expected Behavior
GI should appear on the second scene

###Actual behavior
GI doesn't appear on the second scene

###Configuration
Unity Version: 5.3.5.f1
Bolt Version: 0.4.3.11
Operating System: Windows 10
Network Configuration: Ethernet behind router

**Comment by Darkworth** [comment]
Was having this same issue and found the fix in the unity forums... applied it and it fixed the issue in bolt as well.  (IE doesn't seem to be anything in bolt causing it as normal unity users are having this issue too)

1. This issue is editor only.  Lighting works properly in builds.
2. To fix this in the editor need to go into the lighting tab, and disable auto building of lighting at the bottom and build it manually.  I tested this in a scene that was having the lost lighting issue and both cases were fixed (IE in the built version the lighting worked as expected, and once i disabled auto building of the lighting and built it manually the lighting now worked as expected in the editor as well)

Hope this helps.
